src/main/java/com/enderio/core/common/BlockEnder.java

```diff
@@ -167,6 +168,20 @@ protected boolean shouldDoWorkThisTick(World world, int x, int y, int z, int int
     }
   }
 
+  @Override
+  public boolean hasComparatorInputOverride() {
+    return IComparatorOutput.class.isAssignableFrom(teClass);
```

Comment:
See https://gist.github.com/michaeldorner/9b2fa6eb3711db2ab92c for performance measurements of "instanceof" against an identity check, the call to an overridden method and checking an enum. Result: http://i.stack.imgur.com/pI4Dj.png